### DISCUSSION TITLE
Unable to find module with tsconfig-paths

### DISCUSSION THREAD

**Discussion by edaqa-uncountable** [TypeStrong/ts-node#1270] [Q&A] [answered]
ts-node is unable to find a module and I don't know how to further debug the issue.

I use a command-line like this:
```
node_modules/.bin/ts-node --project ./tsconfig.json -r tsconfig-paths/register test/js/sample.ts
```

Where there is a `paths` section in the tsconfig.ts file
```
  "compilerOptions": {
    "baseUrl": "js",
    "paths": {
      "some_base/*": ["base/*"],
    }
```

And then I try to import via it:

```
// this fails
import { checkConstraints } from 'some_base/comp'

// this works (not using path aliases)
import { checkConstraints } from 'base/comp'
```

I was unable to find an option in ts-node or tsconfig-paths that would let me diagnose the issue further.

**Comment by cspotcode** [comment, marked as answer]
Check this out:
https://github.com/TypeStrong/ts-node/discussions/1263

**Comment by edaqa-uncountable** [comment]
I'm looking for a way to debug the setup you've linked to, since I am using `tsconfig-paths` (see the command-line) but it's unable to resolve the modules.

**Reply by cspotcode** [reply]
If such a mechanism exists, it'll be a feature of tsconfig-paths, so you can check their documentation.

**Reply by edaqa-uncountable** [reply]
Asked at https://github.com/dividab/tsconfig-paths/issues/160
Linking in case people end up here searching.
